python/lsst/afw/geom/ellipses/__init__.py

```diff
@@ -23,23 +23,20 @@
 """lsst.afw.geom.ellipses
 """
 from __future__ import absolute_import
-from builtins import object
 from .ellipsesLib import *
```

Comment:
We do not permit `from <foo> import *` (except in `__init__.py` files) because it makes the code hard to understand and prevents linters from checking symbols.

Please either import explicit symbols by name or use `import ellipsesLib` and `ellipsesLib.<foo>`. This shows up in many .py files in this commit -- some of which you added (I know this one was pre-existing). Please fix all of the .py files that were touched on this ticket.